### PR TITLE
fixed crossbuild for mingw/win64 issue #3361

### PR DESCRIPTION
**Pull request by enwony**

### CODE REVIEW

File: src/modules/module_28503.c [hunk 1]
```diff
@@ -57,7 +57,7 @@ u32 module_pw_min (MAYBE_UNUSED const hashconfig_t *hashconfig, MAYBE_UNUSED con
 
 static u32 polymod_checksum (const u8 *data, const u32 data_len)
 {
-  const u32 CONST[5] = { 0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3 };
+  const u32 consts[5] = { 0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3 };
```
Comment (ventaquil) [inline src/modules/module_28503.c]:
```suggestion
  const u32 CONSTS[5] = { 0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3 };
```

Tiny issue but maybe you should leave uppercase names?

Comment (enwony) [reply src/modules/module_28503.c]: sure, it will be better (iv seen code - it seems to coded this style ofter)

File: src/modules/module_28503.c [hunk 2]
```diff
@@ -73,7 +73,7 @@ static u32 polymod_checksum (const u8 *data, const u32 data_len)
 
       if (bit_set == 0) continue;
 
-      c ^= CONST[j];
+      c ^= consts[j];
```
Comment (ventaquil) [inline src/modules/module_28503.c]:
```suggestion
      c ^= CONSTS[j];
```

File: src/modules/module_28504.c [hunk 3]
```diff
@@ -73,7 +73,7 @@ static u32 polymod_checksum (const u8 *data, const u32 data_len)
 
       if (bit_set == 0) continue;
 
-      c ^= CONST[j];
+      c ^= consts[j];
```
Comment (ventaquil) [inline src/modules/module_28504.c]:
```suggestion
      c ^= CONSTS[j];
```

File: src/modules/module_28504.c [hunk 4]
```diff
@@ -57,7 +57,7 @@ u32 module_pw_min (MAYBE_UNUSED const hashconfig_t *hashconfig, MAYBE_UNUSED con
 
 static u32 polymod_checksum (const u8 *data, const u32 data_len)
 {
-  const u32 CONST[5] = { 0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3 };
+  const u32 consts[5] = { 0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3 };
```
Comment (ventaquil) [inline src/modules/module_28504.c]:
```suggestion
  const u32 CONSTS[5] = { 0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3 };
```

**Review by ventaquil — CHANGES_REQUESTED** [review]
I proposed change in one place and hoped that you will change all other occurrences.

**Comment by enwony** [comment]
> I proposed change in one place and hoped that you will change all other occurrences.

not fast enough to fix em ;) thanx for fix

**Comment by ventaquil** [comment]
@enwony you can link your PR with your issue which cause automatic close after merge. Just add in description `Fixes GH-3361`.

See [Linking a pull request to an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

**Review by ventaquil — APPROVED** [review]

**Pull request merged**